### PR TITLE
fixed the importing problem of get_all_known_facts and get_known_fact…

### PR DESCRIPTION
**Pull request by MohitKumar020291**
NO ENTRY

### CODE REVIEW

**Comment by MohitKumar020291** [comment]
mistakenly I have not checked approvals as I am new to open source.

**Comment by 1e9abhi1e10** [comment]
> mistakenly I have not checked approvals as I am new to open source.

Should reference the issue in the OP or click [here](https://github.com/sympy/sympy/wiki/Writing-Release-Notes) to see Instructions for writing the release notes.
If you are new then should add your name and email address in the .mailmap file according to the following [Instructions](https://github.com/sympy/sympy/wiki/Development-workflow#add-your-name-and-email-address-to-the-mailmap-file)

**Comment by MohitKumar020291** [comment]
done release notes with no entry as it is my first contribution

**Comment by MohitKumar020291** [comment]
I am unable to get approval from maintainer . Kindly tell me how to get it.

File: sympy/assumptions/ask.py [hunk 1]
```diff
@@ -628,5 +628,7 @@ def remove_handler(key, handler):
         getattr(Q, key).remove_handler(handler)
 
 
-from sympy.assumptions.ask_generated import (get_all_known_facts,
-    get_known_facts_dict)
+from sympy.assumptions.ask_generated import (get_all_known_facts)
+from sympy.assumptions.ask_generated import ( get_known_facts_dict)
+
+
```
Comment (oscarbenjamin) [inline sympy/assumptions/ask.py:634]: I don't understand why this is being changed but the change does not do anything useful.

Apart from this change there are two commits that delete files that should not be deleted.

**Comment by oscarbenjamin** [comment]
I'm not going to authorise the workflow because I can tell already that the tests will immediately fail. I don't see anything useful here and I don't understand what problem this is supposed to fix.

I'm going to close this.

If there is an issue that this is supposed to fix then please reference that or comment in the issue.

**Comment by MohitKumar020291** [comment]
When I ran the above line that are actually changed by me are showing error so , I changed it. Now after implementing it I did not find any error. I also want to know that how find real problems that works and helps the community.

**Pull request closed without merging**